src/MoBi.UI/Views/EditContainerView.cs

```diff
@@ -70,13 +80,20 @@ public override void InitializeResources()
          btName.ToolTip = ToolTips.Container.ContainerName;
          tabProperties.InitWith(AppConstants.Captions.Properties, ApplicationIcons.Properties);
          tabParameters.InitWith(AppConstants.Captions.Parameters, ApplicationIcons.Parameter);
+         layoutItemParentPath.Text = AppConstants.Captions.ParentPath.FormatForLabel();
+         layoutControl.InitializeDisabledColors(_lookAndFeel);
```

Comment:
This lines ensure that disable fields TEXT is not grayed out butg readble

Reply:
finally :+1: